File: templates/Uwp/_comp/Prism/Page._NoParams.UnitTests.App.MSTest/Param_ProjectName.Tests.MSTest/Tests$wts.ItemName_gpostaction.cs
//{[{
using Param_RootNamespace.ViewModels;
//}]}

namespace Param_RootNamespace.Tests.MSTest
{
    public class Tests
    {
        //^^
        //{[{

        // TODO WTS: Add tests for functionality you add to wts.ItemNameViewModel.
        [TestMethod]
        public void Testwts.ItemNameViewModelCreation()
        {
            // This test is trivial. Add your own tests for the logic you add to the ViewModel.
            var vm = new wts.ItemNameViewModel();
            Assert.IsNotNull(vm);
        }
        //}]}
    }
}

File: code/TemplateStudioForUWP/Templates/_comp/_shr/Page.MSTest._VB/Param_ProjectName.Tests.MSTest/Tests$wts.ItemName_gpostaction.vb
'{[{
Imports Param_RootNamespace.ViewModels
'}]}

Public Class Tests
    '^^
    '{[{

    ' TODO: Add tests for functionality you add to wts.ItemNameViewModel.
    <TestMethod>
    Public Sub Testwts.ItemNameViewModelCreation()
        ' This test is trivial. Add your own tests for the logic you add to the ViewModel.
        Dim vm = new wts.ItemNameViewModel()
        Assert.IsNotNull(vm)
    End Sub
    '}]}
End Class
